gflags.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: gflags
+  version: 2.2.2
+  epoch: 0
+  description: "The gflags package contains a C++ library that implements commandline flags processing."
+  copyright:
+    - license: "BSD-3-Clause"
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - cmake
+      - build-base
+      - samurai
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/gflags/gflags/
+      tag: v${{package.version}}
+      expected-commit: e171aa2d15ed9eb17054558e0b3a6a413bb01067
+
+  - uses: cmake/configure
+    with:
+      opts: |
+        -DCMAKE_BUILD_TYPE=None \
+        -DBUILD_STATIC_LIBS=ON \
+        -DBUILD_SHARED_LIBS=ON \
+        -DBUILD_TESTING=ON \
+        -DCMAKE_INSTALL_PREFIX=/usr \
+        -DREGISTER_INSTALL_PREFIX=OFF \
+        -G Ninja
+
+  - uses: cmake/build
+
+  - uses: cmake/install
+
+  - uses: strip
+
+subpackages:
+  - name: "gflags-dev"
+    description: "The gflags package contains a C++ library that implements commandline flags processing (development files)"
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      provides:
+        - glog-dev=${{package.full-version}}
+
+update:
+  enabled: true
+  github:
+    identifier: gflags/gflags
+    use-tag: true
```

Comment:
```suggestion

```
we can also remove this as the gflags repo uses GitHub releases which is a preferred way to get that associated tag https://github.com/gflags/gflags/releases

Reply:
done, thanks!